src/App.js

```diff
@@ -25,12 +26,18 @@ function App() {
                 <Route
                   exact
                   path="/"
-                  render={() => <Shopping list={data} userToken={userToken} />}
+                  render={() => <Shopping userToken={userToken} list={data} />}
                 />
                 <Route
                   path="/add"
                   render={() => <AddItem userToken={userToken} list={data} />}
                 />
+                <Route
+                  path="/detail/:itemId"
+                  render={() => (
+                    <ItemDetail userToken={userToken} list={data} />
+                  )}
+                />
                 <nav id="nav">
```

Comment:
It's odd having this navigation on the Details View. I think of this view as a pop-up with extra information. You have the back arrow to the Shopping view, and that is a good feature. But I think to remove this you would have to rework some of the other pages code and make a Nav component.

Reply:
Good call out. I was thinking the same thing.